lib/qam.pm

```diff
@@ -136,10 +140,17 @@ sub get_patches {
     $repo =~ tr/,/ /;
 
     # Search for patches by incident, exclude not needed
-    my $patches = script_output("zypper patches -r $repo | awk -F '|' '/[Nn]eeded/ && !/[Nn]ot [Nn]eeded/ && /$incident_id/ { printf \$2 }'");
-    # Remove carriage returns and make patch list on one line
-    $patches =~ s/\r//g;
-    return $patches;
+    my $patches = script_output("zypper patches -r $repo");
+    my @patch_list;
+
+    for my $line (split /\n/, $patches) {
+        my @tokens = split /\s*\|\s*/, $line;
+        next if $#tokens < max(ZYPPER_PACKAGE_COL, ZYPPER_STATUS_COL);
+        push @patch_list, $tokens[ZYPPER_PACKAGE_COL]
+          if 'needed' eq lc $tokens[ZYPPER_STATUS_COL];
```

Comment:
`[Nn]eeded` was changed to `needed`, does it mean that we don't need `Needed` anymore?

Reply:
> `[Nn]eeded` was changed to `needed`, does it mean that we don't need `Needed` anymore?

i'm blind ignore my comment